docs/account_management/cheaha_account.md

```diff
@@ -23,6 +23,33 @@ When you are ready, click "Create Account" to start the account creation process
 
 Welcome to Cheaha and to Research Computing!
 
+## Cheaha Account and Group Membership FAQ
```

Comment:
Let's move the FAQ to the end of the page.

Reply:
Fixed.